### PR TITLE
Fix alpine missing test-deps

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

* Quit using regex tag mappings for alpine test-dep
* Update centos7 image sizes
* Update to new image from credscan
* Only use first LTS ReleaseTag

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [ ] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/docs/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: release/lts/centos7/test-deps/meta.json [hunk 1]
```diff
@@ -10,6 +10,6 @@
         "test-deps"
     ],
     "TestProperties": {
-        "size": 575
+        "size": 610
```
Comment (RDIL) [inline release/lts/centos7/test-deps/meta.json:13]: Anything in particular that caused these jumps?

**Review by RDIL — APPROVED** [review]

File: release/lts/centos7/test-deps/meta.json (hunk 1 above)
Comment (TravisEz13) [reply release/lts/centos7/test-deps/meta.json:13]: probably just 7.2... there is also a 20% grace in the test... so, it takes a bit before we must bump the value.

**Comment by TravisEz13** [comment]
@RDIL  thanks for the review.

File: release/lts/centos7/test-deps/meta.json (hunk 1 above)
Comment (TravisEz13) [reply release/lts/centos7/test-deps/meta.json:13]: There were some cases where images had grown by large amounts. say powershell was put on the image twice... that's what we are looking for... "natural" growth doesn't worry me.

**Pull request merged**